player/lua/defaults.lua

```diff
@@ -1,5 +1,5 @@
 -- Compatibility shim for lua 5.2/5.3
-unpack = unpack or table.unpack
+local unpack = unpack or table.unpack -- luacheck: globals unpack table.unpack
```

Comment:
This makes it inaccessible from other scripts.

Reply:
It is not used in any other scripts, so it is made local. It is not something we should export anyway.

Reply:
I don't know if 3rd-party scripts may depend on it. But according to https://github.com/mpv-player/mpv/issues/1648 it is only needed if you compile 5.2 without `LUA_COMPAT_UNPACK` anyway. I did use this in the select.lua PR so I'll update that.

Reply:
We can export it as mp.unpack.

> I don't know if 3rd-party scripts may depend on it.

Well, not our problem. Though indeed, we would silently fix global unpack if it were to be not available.

Reply:
It's probably fine to just make it local with how rarely it is needed.